### PR TITLE
Add OQMD exercises to repository

### PR DESCRIPTION
**Pull request by ml-evs**
As written by @tachyontraveler.

I added the data in a subdir to avoid cluttering the `notebooks` dir, but the filepaths should have also been updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -395,19 +401,19 @@ prefix to the keyword: `_aflow_aflow_prototype_label_relax`.
...
+This interactive exercise explores the OQMD's OPTIMADE API, and
+demonstrates how you can train machine learning models on OPTIMADE data.
+The notebook is available at
+`./notebooks/exercise7-oqmd-optimade-tutorial` and can also be accessed
 online with
-[Colab](https://colab.research.google.com/github/tachyontraveler/oqmd-optimade-tutorial/blob/main/Workbook.ipynb)
+[Colab](https://colab.research.google.com/github/Materials-Consortia/optimade-tutorial-exercises/blob/main/notebooks/exercise7-oqmd-optimade-tutorial.ipynb)
 or
-[Binder](https://mybinder.org/v2/gh/tachyontraveler/oqmd-optimade-tutorial/HEAD?filepath=Workbook.ipynb).
+[Binder](https://mybinder.org/v2/gh/Materials-Consortia/optimade-tutorial-exercises/HEAD?filepath=notebooks/exercise7-oqmd-optimade-tutorial.ipynb)
```
Comment (JPBergsma) [inline README.md:411]: When I click on the binder button I get an error:

 404 : Not Found
You are requesting a page that does not exist!

Comment (ml-evs) [reply README.md:411]: That link won't work until this PR has merged.

Comment (JPBergsma) [reply README.md:411]: Ah, yes you are right. When I clicked on the button I got redirected to a binder page and from the address is was not clear it loaded the page fromn the git hub.

File: notebooks/exercise7-oqmd-optimade-tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,645 @@
...
+    "\n",
+    "load_data_from_saved = True  \n",
+    "# This is to avoid querying OQMD repeatedly for the same data, if .\n",
+    "# Because the data I downloaded is already available as a JSON file in this Git repo.\n",
+    "\n",
+    "# But if you'd like to try out querying OQMD, set \"load_data_from_saved\" as \"False\"\n",
+    "# Querying OQMD for this particular data would take about 5-10 minutes to complete\n",
+    "\n",
+    "\n",
+    "dataset_filename = Path(__file__).parent.joinpath(\"./oqmd_exercise_data/raw_dataset_saved.json\")\n",
```
Comment (JPBergsma) [inline notebooks/exercise7-oqmd-optimade-tutorial.ipynb]:
```suggestion
"if \"__file__\" in globals():\n",
    "    dataset_filename = Path(__file__).parent.joinpath(\"./oqmd_exercise_data/raw_dataset_saved.json\")\n",
    "else:\n",
    "    dataset_filename = Path(\"__file__\").parent.joinpath(\"./oqmd_exercise_data/raw_dataset_saved.json\")\n",
    "    \n",
```
\_\_file\_\_ does not seem to work in a jupyter notebook.
And I get the error message: 
NameError: name '\_\_file\_\_' is not defined

Replacing it with "\_\_file\_\_" does work.  To not break the original code I added an if statement to distinguish between these two situations.
See also [here](https://stackoverflow.com/questions/63583062/path-problem-nameerror-name-file-is-not-defined)

File: notebooks/exercise7-oqmd-optimade-tutorial.ipynb [hunk 3]
```diff
@@ -0,0 +1,645 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load the data generated in previous sections\n",
+    "\n",
+    "# Input feature data - generated using Magpie on POSCAR files \n",
+    "features = open(\"./features_processed_final.csv\",\"r\").read().strip().split(\"\\n\")\n",
```
Comment (JPBergsma) [inline notebooks/exercise7-oqmd-optimade-tutorial.ipynb:440]:
```suggestion
    "if os.path.isfile(\"./features_processed_final.csv\"): # Case where the optional part has been done\n",
    "    features = open(\"./features_processed_final.csv\",\"r\").read().strip().split(\"\\n\")\n",
    "else: \n",
    "    features = open(\"./oqmd_exercise_data/features_processed_final.csv\",\"r\").read().strip().split(\"\\n\") # When the optional part was not done\n",
```
If they have not done the optional part, the file: "features_processed_final.csv" is not present in the directory where the Jupyter notebook is stored, and they instead need to load it from the oqmd_exercise_data folder

**Comment by ml-evs** [comment]
Thanks @JPBergsma, I have made your suggested fixes. I think this is ready to merge now, so people have it in time for the tutorials tomorrow.

**Comment by JPBergsma** [comment]
I tried the new version, but the paths were still not correct, I have created PR#5 to fix this.

**Pull request merged**